bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>before this test, there shall be a "preview" of the story saying that the next one won't answer about uber as the filter only allows lyft related content to be retrieved. While the next qusetion can be answer as it's related to lyft. We don't have to let users crack the puzzle and guess that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>